lntest/fee_service.go

```diff
@@ -114,9 +119,12 @@ func (f *FeeService) handleRequest(w http.ResponseWriter, _ *http.Request) {
 
 	bytes, err := json.Marshal(
 		struct {
-			Fees map[uint32]uint32 `json:"fee_by_block_target"`
+			//nolint:lll
+			Fees            map[uint32]uint32     `json:"fee_by_block_target"`
+			MinRelayFeerate chainfee.SatPerKVByte `json:"min_relay_feerate"`
```

Comment:
could import the struct right? (since it is no not anonymous anymore and is exported) - then dont need to re-type the json tags here

Reply:
totally!